src/show.jl

```diff
@@ -50,6 +59,7 @@ function print_graded_results(io::IO, res::GradingResult, depth_pad=0)
 end
 
 function _category_width(category_name, count)
+    count = round(Int, count)
     digits = count > 0 ? ndigits(count) : 0
     return digits > 0 ? max(length(category_name), digits) : 0
 end
```

Comment:
This function is wrong. It makes sure the columns in the printout are wide enough to fit the number. If a student has `8.5` points, the width `digits` is 3. If you round `8.5` to an integer, die width is changed to 1.

Just use `digits = length(string(count))`.

Reply:
You're right. I think this was still a remnant from my thinking process of transforming numbers to `Int` if they had no decimal places.